bcgov/moh-prime
# PRIME-2183 Can't navigate from CareSettings page in paper enrolment

[PRIME-2183](https://bcgovmoh.atlassian.net/browse/PRIME-2183)

### How to test:

1. Start a paper enrolment
2. On the Care Settings page, select Health Authority, then select another Care Setting
3. Press "Save and Continue" (You should be able to navigate to next page)
